AppCreationScripts/createdApps.html

```diff
@@ -0,0 +1,4 @@
+<html><body><table>
```

Comment:
You don't want to commit this one.

Reply:
Removed a bunch of files. It was a mistake.